### PR TITLE
TRUS-477 | Created agentanswers page, added test and sections to view

### PR DESCRIPTION
**Pull request by JoshuaH01**

### CODE REVIEW

File: test/navigation/navigators/AgentRoutes.scala [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import navigation.Navigator
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalatest.prop.PropertyChecks
+import pages._
+
+trait AgentRoutes {
+
+  self: PropertyChecks with Generators with SpecBase =>
+
+  def agentRoutes()(implicit navigator: Navigator) = {
```
Comment (adamconder) [inline test/navigation/navigators/AgentRoutes.scala]: You need to call this from NavigatorSpec and implement `behave like agentRoutes()`

Comment (JoshuaH01) [reply test/navigation/navigators/AgentRoutes.scala]: Fixed

File: app/controllers/AgentAnswerController.scala [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  def onPageLoad= actions {
+    implicit request =>
+
+      val checkYourAnswersHelper = new CheckYourAnswersHelper(countryOptions)(request.userAnswers)
+
+      val sections = Seq(
+        AnswerSection(
+          None,
+          Seq(
+            checkYourAnswersHelper.agenciesTelephoneNumber
```
Comment (adamconder) [inline app/controllers/AgentAnswerController.scala]: When we merge this into feature branch we will need to update CYA as well as navigator.

File: test/views/AgentAnswerViewSpec.scala [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import views.behaviours.ViewBehaviours
+import views.html.AgentAnswerView
+
+class AgentAnswerViewSpec extends ViewBehaviours {
+
+  "AgentAnswer view" must {
+
+    val application = applicationBuilder(userAnswers = Some(emptyUserAnswers)).build()
+
+    val view = application.injector.instanceOf[AgentAnswerView]
```
Comment (adamconder) [inline test/views/AgentAnswerViewSpec.scala]: Please use `viewFor[T]` instead of `application.injector.instanceOf[T]`

**Pull request merged**